index.html

```diff
@@ -51,9 +51,220 @@ <h2 class="hero-msg">Glad to see you!</h2>
+              <img src="./img/Portfolio3.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>Tonic</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card-container">
+        <div class="card-wrapper">
+          <div class="card">
+            <div class="card-image">
+              <img src="./img/Portfolio4.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>Multi-Post Stories</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <main>
+      <div class="hero">
+        <div>
+          <h1 class="hero-name">About</h1>
```

Comment:
- [ ] Considering the fact you've used a `h1` tag for the intro in the previous implantation above, kindly note that it's semantically a good practice to have only one `h1` element in a whole project.

- [ ]  Using only one `h1` tag in an HTML file is recommended for two main reasons:

- Accessibility: Screen readers use the heading structure of a webpage to provide an overview of the content and navigate through it. If there are multiple `h1` tags on a page, it can create confusion and make it difficult for screen readers to provide a clear and logical hierarchy of the content.

- SEO: Search engines also use the heading structure of a webpage to understand its content and rank it in search results. Having multiple `h1` tags can dilute the importance and relevance of each heading, making it more difficult for search engines to determine the main topic or focus of the page.